neo4j/session_with_context.go

```diff
@@ -398,7 +408,7 @@ func (s *sessionWithContext) executeTransactionFunction(
 		return true, nil
 	}
 
-	s.retrieveBookmarks(conn, beginBookmarks)
+	s.retrieveBookmarks(nil, conn, beginBookmarks)
```

Comment:
Why is the context being omitted here?

Reply:
oops, strange oversight

Reply:
Fixed in e2299cc5f46f2f6acd623af39a96bc950994ef70